oteapi/models/datacacheconfig.py

```diff
@@ -36,6 +38,5 @@ class DataCacheConfig(AttrDict):
     tag: Optional[str] = Field(
         None,
         description="Tag assigned to the downloaded content, typically "
-        "identifying a session. Used with the `evict()` method to clean up a "
-        "all cache entries with a given tag.",
+        "identifying a session. Used with the `evict()` method to clean up all cache entries with a given tag.",
```

Comment:
Please ahead to the [PEP8](https://peps.python.org/pep-0008/#maximum-line-length) Python conventions. Max line length is 80 characters.